crypto/threads_pthread.c

```diff
@@ -59,7 +59,11 @@ __tsan_mutex_post_lock((x), 0, 0)
 
 # include <assert.h>
 
-# ifdef PTHREAD_RWLOCK_INITIALIZER
+/*
+ * The Non-Stop KLT thread model currently seems broken in its rwlock
+ * implementation
+ */
+# if defined(PTHREAD_RWLOCK_INITIALIZER) && !defined(_KLT_MODEL_)
```

Comment:
Is this part missing from 3.3 and eralier?

Reply:
thats probably a question for @rsbecker .  @rsbecker was there a reason that the KLT changes didn't go to 3.3 and earlier?